README.md

```diff
@@ -43,10 +43,17 @@ Example configuration file:
 LOG_LEVEL_MAIN = INFO
 RUN_FLAKE8 = False
 RUN_ANSIBLE_TEST = False
-INFRA_PULP = False
+INFRA_LOCAL_IMAGE = False
+INFRA_LOCAL_DOCKER = False
 INFRA_OSD = False
 ```
 
+- `RUN_ANSIBLE_TEST` - Set to `True` to run `ansible-test` during collection import. If `True`, requires Ansible to be installed, unless container image execution is enabled. Defaults to `False`.
+
+- `INFRA_LOCAL_IMAGE` - Set to `True` to run `ansible-test` sandboxed within a container image. Requires installation of either Podman or Docker to run the container. Defaults to `False`.
```

Comment:
I think a config value that denotes when to run ansible-test in local container is good. Name suggestion: `ansible_test_local_image` and `local_image_docker` (originally infra* was denoting things like if backend was openshift or not, but that convention might not be very useful after all)

Reply:
Sounds good to me.

Reply:
We should remove `If 'True', requires Ansible to be installed, unless container image execution is enabled.` - as ansible is required by the package in general for ansible-doc and ansible-lint